### ISSUE TITLE
support the "| drop" pipeline for grafana 10.3+

### ISSUE THREAD

**Issue by deathalt** [metrico/qryn#450]
```
{"level":30,"time":1706794482302,"pid":23,"hostname":"qryn-read-6656b4bc59-82cxj","name":"qryn","reqId":"req-e","req":{"method":"GET","url":"/loki/api/v
1/query_range?direction=backward&end=1706794500000000000&query=sum+by+%28level%29+%28count_over_time%28%7Bjob%3D%22service%22%2C+level%3D~
%22ERROR%22%7D+%7C+json+Environment%3D%22resources%5B%5C%22deployment.environment%5C%22%5D%22%2C+body%3D%22body%22+%7C+Environment+%3D+%60Stage%60+%7C+d
rop+__error__%5B1m%5D%29%29&start=1706790840000000000&step=60000ms","hostname":"qryn-read.qryn.svc:3100","remoteAddress":"172.24.22.31","remotePort":487
56},"msg":"incoming request"}                                                                                                                           
Couldn't execute script:                                                                                                                                
Expected [ or 'SYNTAX' ] @1:0 encountered 'sum b'.                                                                                                      
sum by (level) (count_over_time({job="service", level=~"ERROR"} | json Environment="resources[\"deployment.environment\"]", body="body" | 
Environment = `Stage` | drop __error__[1m]))                                                                                                            
{"level":50,"time":1706794482322,"pid":23,"hostname":"qryn-read-6656b4bc59-82cxj","name":"qryn","reqId":"req-e","err":"invalid request\nBad Request: inv
alid request\n    at module.exports.transpile (/app/parser/transpiler.js:141:11)\n    at scanFingerprints (/app/lib/db/clickhouse.js:242:29)\n    at Obj
ect.handler (/app/lib/handlers/query_range.js:41:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","msg":"invalid 
request"}       
```
logs themselves are loaded, but logs volume (where logs are distributed by time) are not given by the error "Failed to load log volume for this query
invalid request"

for logs volume aggregation grafana make request like
```
sum by (level) (count_over_time({job="service", level=~"ERROR"} | json Environment="resources[\"deployment.environment\"]", body="body" | 
Environment = `Stage` | drop __error__[1m])) 
```

and its fails on qryn <= 3.2.1
![image](https://github.com/metrico/qryn/assets/35489816/483b37ab-6b71-4aff-9871-864fe1f9ff1e)

**Comment by akvlad** [comment]
@deathalt should be fixed in 3.2.2

**Comment by deathalt** [comment]
Fixed
